### PR TITLE
Ownership update on network policies

### PR DESCRIPTION
**Pull request by jotak**
Proposal of shared ownership (console team and network observability
team), as well as new reviewers on files related to
network policies (note: it doesn't include, at the moment, the policies
list / details page that lies in another subtree)

Also adding myself as an approver for the locales subtree

It is motivated by a recent involvment by the network observability team
to add features related to the user experience about network policies
(adding a creation form...) and more future work planned.

Signed-off-by: Joel Takvorian <jtakvori@redhat.com>

### CODE REVIEW

File: frontend/packages/console-app/locales/OWNERS
Comment (spadgett) [inline frontend/packages/console-app/locales/OWNERS:11]: We don't need to add everyone to the reviewers/approvers here. Approvers are inherited from the parent OWNERS file. I would add @rebeccaalpert as a review here and just yourself as an approver.

File: frontend/packages/console-app/src/__tests__/network-policies/OWNERS
Comment (spadgett) [inline frontend/packages/console-app/src/__tests__/network-policies/OWNERS:8]: You can remove me as an approver since I'm an approver in the root OWNERS file for the repo.

File: frontend/packages/console-app/src/components/network-policies/OWNERS
Comment (spadgett) [inline frontend/packages/console-app/src/components/network-policies/OWNERS:8]: You can remove me as an approver here as well.

File: frontend/packages/console-app/src/__tests__/network-policies/OWNERS
Comment (spadgett) [inline frontend/packages/console-app/src/__tests__/network-policies/OWNERS:8]: You'll need to add the label to openshift/release so it's available in the repo.

https://github.com/openshift/release/blob/master/core-services/prow/02_config/_labels.yaml#L548-L549

**Review by spadgett — APPROVED** [review]
/lgtm

File: frontend/packages/console-app/src/__tests__/network-policies/OWNERS
Comment (jotak) [reply frontend/packages/console-app/src/__tests__/network-policies/OWNERS]: Oh, then I've removed it to not block this PR. I'm not sure if this is really something we need.

**Review by spadgett — APPROVED** [review]
/lgtm

**Comment by spadgett** [comment]
/label px-approved
/label docs-approved
/label qe-approved

This is an OWNERS file change only.

**Pull request merged**
